protocols/request-response/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-request-response"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Generic Request/Response Protocols"
-version = "0.28.0"
+version = "0.28.1"
```

Comment:
No version bump needed, `libp2p-request-response-v0.28.0` isn't released yet :)

Reply:
Noticed not to long ago while I was scanning the code :) Not sure what made me think 0.28.0 was released when i checked earlier (might be time to add that suffix :) )